src/index.html

```diff
@@ -1,12 +1,433 @@
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#foto" class="nav__link">
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              VIDEO
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="main-content">
+      <article class="container-section">
+        <section class="section">
+          <div class="card">
+            <div class="card__item" data-qa="card">
+              <img 
+                src="#" 
+                alt="photo-imac" 
+                class="card__image">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p 
+                class="card__product-code"
+                >Product code: 195434
+              </p>
+              <div class="feedback">
+                <div class="list-stars">
+                  <div  class="list-stars__star"></div>
+
+                  <div  class="list-stars__star"></div>
+
+                  <div  class="list-stars__star"></div>
+
+                  <div  class="list-stars__star"></div>
+
+                  <div  class="list-stars__star"></div>
+                </div>
+                <p class="feedback__count">Reviews: 5</p>
+              </div>
+              <p class="card__price">Price: <span class="card__price-digit card__price-digit--bold">$2,199</span></p>
+              <a href="#" data-qa="card-hover">
+                <button class="card__button-buy card__button-buy--hover">BUY</button>
+              </a>
+            </div>
+          </div>
+        </section>
+
+        <section class="section">
+          <div class="card">
+            <div class="card__item">
+              <img
+                src="#"
+                alt="photo-imac" 
+                class="card__image">
+              <h3 class="card__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h3>
+              <p 
+                class="card__product-code"
+                >Product code: 195434
+              </p>
+              <div class="feedback">
+                <div class="list-stars">
+
+                  <div  class="list-stars__star list-stars__star--active">
+                  </div>
+
+                  <div  class="list-stars__star list-stars__star--active">
+                  </div>
+
+                  <div  class="list-stars__star list-stars__star--active">
+                  </div>
+
+                  <div  class="list-stars__star list-stars__star--active">
```

Comment:
Please fix it everywhere in the code like you did above

Reply:
Ops, sorry!! 
I forgot(